src/domain/Lotto.js

```diff
@@ -0,0 +1,51 @@
+import { generateRandomNumber } from "../utils/generateRandomNumber.js";
+
+export class Lotto {
+  static LOTTO_NUMBER_MIN = 1;
+  static LOTTO_NUMBER_MAX = 45;
+  static LOTTO_NUMBER_COUNT_MAX = 6;
```

Comment:
```suggestion
  static Range = { MIN: 1, MAX: 45 }
```
이렇게 표현하면 어떨까요!?

Lotto.LOTTO_NUMBER_MIN 이런식으로 표현하면 중복 표현인데, Lotto.Range.MIN 이렇게 네임스페이스 형식으로 표현할 수 있을것 같아요!

Reply:
그리고 Lotto가 두 가지 역할을 하고 있어요.
LottoNumber 라는 객체도 필요하지 않을까요!?
LottoNumber.MIN
LottoNumber.MAX 
이런식으로 표현할 수 있으면 좋을 것 같아요!

숫자 하나에 대한 검증을 LottoNumber 에게 위임하는거죠 ㅎㅎ